Public/Src/FrontEnd/Core/FrontEndHostController.Nuget.cs

```diff
@@ -723,16 +723,7 @@ public async Task<Possible<string>> DownloadFile(LoggingContext loggingContext,
                 {
                     return LogAndReturnDownloadFailure(e);
                 }
-#if !DISABLE_FEATURE_HTTPEXCEPTION
-                catch (HttpException e)

```

Comment:
HttpException not needed?

Reply:
Even full framework libraries now throw `HttpRequestException`